execute.go

```diff
@@ -70,16 +69,12 @@ func executeWithOutputChan(outputChan chan string, env []string, workingDir stri
 		return nil, fmt.Errorf("error starting process: %v", startErr)
 	}
 
-	waitErr := cmd.Wait()
+	//create a new effectiveconfig instance
+	var con = new(effectiveConfig)
```

Comment:
but if they've changed the value with a config file it does not give you taht value.  which is probably what they want.